CMakeLists.txt

```diff
@@ -355,7 +355,9 @@ endif()
 
 set(TERRIER_LINK_LIBS
     ${TERRIER_LINK_LIBS}
+    gflags
```

Comment:
Why do we need this twice?

Reply:
Some unrelated files are changed will merging cache-pointer branch. Reverting back in process.